orderbook/src/main.rs

```diff
@@ -2,11 +2,18 @@ mod api;
 mod orderbook;
 
 use crate::orderbook::OrderBook;
+use ethcontract::{Http, Web3};
+use model::DomainSeparator;
+use primitive_types::H160;
+use std::fs;
 use std::{net::SocketAddr, sync::Arc, time::Duration};
 use structopt::StructOpt;
 use tokio::task;
 use warp::Filter;
 
+#[path = "../../contracts/src/paths.rs"]
```

Comment:
I did not know how to do it correctly. 

Should we share this code in the lib of contracts?

Gernerally, I am not sure whether we really wanna read the contract addresses from here. I will wait for Valentins implementation to see "a proper solution"

Reply:
You should be able to get the address from calling `GPv2Settlement::deployed().await`